keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -262,6 +262,7 @@ open class SqlResourceRepository(
 
       txn
         .insertInto(EVENT)
+        .set(EVENT.UID, ULID().nextULID(event.timestamp.toEpochMilli()))
```

Comment:
Ensure we assign `uid` to new events

Reply:
I hadn't seen an example initializing the ULID based on a timestamp before. Neat!

Reply:
Yeah, the nice thing is that because I am doing the same on the old events, we can sort by `uid` and they will be in timestamp order

Reply:
Awesome!